plugins/chunter-resources/src/components/EditChannelDescriptionAttachments.svelte

```diff
@@ -0,0 +1,154 @@
+<!--
+//
+// Copyright © 2022 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import attachment, { Attachment } from '@anticrm/attachment'
+  import { AttachmentPresenter } from '@anticrm/attachment-resources'
+  import { Channel } from '@anticrm/chunter'
+  import type { Doc } from '@anticrm/core'
+  import { createQuery } from '@anticrm/presentation'
+  import { Menu } from '@anticrm/view-resources'
+  import { showPopup, IconMoreV, Label } from '@anticrm/ui'
+
+  export let channel: Channel | undefined
+
+  const query = createQuery()
+  let attachments: Attachment[] | undefined
+  let selectedRowNumber: number | undefined
+  let attachmentsLimit: number | undefined = 5
+
+  const showMenu = async (ev: MouseEvent, object: Doc, rowNumber: number): Promise<void> => {
+    selectedRowNumber = rowNumber
+    showPopup(Menu, { object }, ev.target as HTMLElement, () => {
+      selectedRowNumber = undefined
+    })
+  }
+
+  $: channel &&
+    query.query(
+      attachment.class.Attachment,
+      {
+        space: channel._id
+      },
+      (res) => {
+        attachments = res
+      },
+      attachmentsLimit ? { limit: attachmentsLimit } : undefined
+    )
+</script>
+
+<div class="group">
+  <div class="eGroupTitle"><Label label={attachment.string.Files} /></div>
+  {#if attachments?.length}
+    <div class="flex-col">
+      {#each attachments as attachment, i}
+        <div class="flex-between attachmentRow" class:fixed={i === selectedRowNumber}>
+          <div class="item flex">
+            <AttachmentPresenter value={attachment} />
+          </div>
+          <div class="eAttachmentRowActions" class:fixed={i === selectedRowNumber}>
+            <div id="context-menu" class="eAttachmentRowMenu" on:click={(event) => showMenu(event, attachment, i)}>
+              <IconMoreV size={'small'} />
+            </div>
+          </div>
+        </div>
+      {/each}
+      {#if attachmentsLimit && attachments.length === attachmentsLimit}
```

Comment:
If we have only 5 attachments, we have this button... Could you please watch https://github.com/hcengineering/anticrm/pull/1399/files#diff-5029dbbcc5e1d0d2b452edcf85a80ff7db8a9c644b714d2607d95f1b4d9051fcR82 for example